packages/form-layout/src/vaadin-form-layout.js

```diff
@@ -433,18 +437,33 @@ class FormLayout extends ElementMixin(ThemableMixin(mixinBehaviors([IronResizabl
   }
 
   /** @private */
-  _invokeUpdateStyles() {
-    this.updateStyles();
+  _invokeUpdateLayout() {
+    this.__updateLayout();
   }
 
   /**
    * Set custom CSS property values and update the layout.
-   * @param {Object<string, string>=} properties
-   * @protected
+   *
+   * @deprecated Since Vaadin 23, `updateStyles()` is deprecated.
+   * Use the native element.style.setProperty API to set custom CSS property values.
    */
   updateStyles(properties) {
-    super.updateStyles(properties);
+    console.warn(
+      `WARNING: Since Vaadin 23, updateStyles() is deprecated. Use the native element.style.setProperty API to set custom CSS property values.`
+    );
+
+    for (const p in properties) {
```

Comment:
nit: should we use `properties.forEach` instead?

Reply:
The implementation was derived from [here](https://github.com/webcomponents/polyfills/blob/2398e9ddb3241855a66938c6969e9f3cb09eed48/packages/shadycss/src/common-utils.js#L21-L27). Changed to use `Object.entries(...).forEach` instead.